xml/docu_styleguide.terminology.xml

```diff
@@ -886,7 +897,9 @@
       <entry>list box</entry>
       <entry>list, list field</entry>
       <entry>noun; GUI element that is a list showing multiple elements even
-              before interacting with it
+              before interacting with it. To help the user interact with UI,
+              refer to a list box by its label and the word
+              <emphasis>list</emphasis>, not <emphasis>list box</emphasis>.
```

Comment:
I don't think we need the last sentence as it has little to do with terminology. If we refer to drop-downs by label, we should do so consistently for other widgets as well. Whether or not to use labels is a different question (and one we should certainly discuss in a separate issue).